app/assets/javascripts/discourse/app/components/composer-toggles.js

```diff
@@ -29,13 +29,13 @@ export default class ComposerToggles extends Component {
   toggleIcon(composeState) {
     return composeState === "draft" || composeState === "saving"
       ? "xmark"
-      : "chevron-down";
+      : "angles-down";
   }
 
   @discourseComputed("composeState")
   fullscreenIcon(composeState) {
     return composeState === "draft"
-      ? "chevron-up"
+      ? "angles-up"
```

Comment:
I would have picked the chevron as the default and used that in chat. Mostly from the perspective of opting to change the icon in the less travelled user path to match the current standard. Changing it in the composer is a much bigger "cheese move" for virtually all users (vs in chat, used by a smaller subset of users/instances).

Reply:
This was the choice of chapoi in the internal discussion, you should rather say it here